.github/workflows/binderbot.yaml

```diff
@@ -15,7 +15,7 @@ on:
     branches:
       - main
     paths:
-      - 'notebooks/Run-Anywhere/*.ipynb'
+      #- 'notebooks/Run-Anywhere/**/*.ipynb'
       - 'binder-gallery.yaml'
       - 'thumbnail.png'
       - '.github/workflows/binderbot.yaml'
```

Comment:
Same thing here

Reply:
OK thanks!  I think I should close this PR and try again on a new branch.